doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -317,11 +317,18 @@ Cellular samples (renamed from nRF9160 samples)
 * :ref:`nrf_cloud_multi_service` sample:
 
   * Renamed Cellular: nRF Cloud MQTT multi-service to Cellular: nRF Cloud multi-service.
-  * Added documentation for using the :ref:`lib_nrf_cloud_alert` and :ref:`lib_nrf_cloud_log` libraries.
-  * Changed the :file:`overlay_nrfcloud_logging.conf` file to enable JSON logs by default.
-  * The :c:struct:`nrf_cloud_obj` structure and associated functions are now used to encode and decode nRF Cloud data.
-  * Cleaned up and simplified connection logic.
-  * Switched from using the :ref:`lte_lc_readme` library directly to using Zephyr's ``conn_mgr`` and the :kconfig:option:`CONFIG_LTE_CONNECTIVITY` Kconfig option.
+  * Added:
+
+    * Documentation for using the :ref:`lib_nrf_cloud_alert` and :ref:`lib_nrf_cloud_log` libraries.
+    * An overlay that allows the sample to be used with Wi-Fi instead of LTE.
+
+  * Updated:
+
+    * The :file:`overlay_nrfcloud_logging.conf` file to enable JSON logs by default.
+    * The encoding and decoding of nRF Cloud data to use the :c:struct:`nrf_cloud_obj` structure and associated functions.
+    * The connection logic by cleaning and simplifying it.
+    * From using the :ref:`lte_lc_readme` library directly to using Zephyr's ``conn_mgr`` and the :kconfig:option:`CONFIG_LTE_CONNECTIVITY` Kconfig option.
+
   * Removed the Kconfig options :kconfig:option:`CONFIG_LTE_INIT_RETRY_TIMEOUT_SECONDS` and :kconfig:option:`CLOUD_CONNECTION_REESTABLISH_DELAY_SECONDS` as they are no longer needed.
   * Added the :file:`overlay_coap.conf` file to enable the use of CoAP instead of MQTT to connect with nRF Cloud.
```

Comment:
Since we're going on top, this needs to move to the Added section edited like this:
```suggestion
    * The :file:`overlay_coap.conf` file to enable the use of CoAP instead of MQTT to connect with nRF Cloud.
```